pipeline_dp/budget_accounting.py

```diff
@@ -104,6 +109,64 @@ def request_budget(
     def compute_budgets(self):
         pass
 
+    def scope(self, weight: float):
+        """
```

Comment:
Nit: according to styleguide, function comments

"""short description

long description
"""